doc/api/api_changes.rst

```diff
@@ -341,50 +390,47 @@ The deprecated ``matplotlib.rcsetup.validate_maskedarray``,
 removed.
 
 
-Deprecations
-------------
+The kwarg ``resolution`` of
+:class:`matplotlib.projections.polar.PolarAxes` has been removed. It
+has deprecation with no effect from version `0.98.x`.
 
-- `matplotlib.testing.noseclasses` is deprecated and will be removed in 2.3
 
+``Axes.set_aspect("normal")``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Functions removed from the `lines` module
------------------------------------------
+Support for setting an ``Axes``\'s aspect to ``"normal"`` has been
+removed, in favor of the synonym ``"auto"``.
 
-The `matplotlib.lines` module no longer imports the `pts_to_prestep`,
-`pts_to_midstep` and `pts_to_poststep` functions from the `matplotlib.cbook`
-module.
 
-Invalid (Non-finite) Axis Limit Error
--------------------------------------
+``shading`` kwarg to ``pcolor``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-When using :func:`~matplotlib.axes.Axes.set_xlim` and
-:func:`~matplotlib.axes.Axes.set_ylim`, passing non-finite values now
-results in a ``ValueError``. The previous behavior resulted in the
-limits being erroneously reset to ``(-0.001, 0.001)``.
+The ``shading`` kwarg to `~matplotlib.axes.Axes.pcolor` has been
+removed.  Set ``edgecolors`` appropriately instead.
 
-``scatter`` and ``Collection`` offsets are no longer implicitly flattened
--------------------------------------------------------------------------
 
-`~matplotlib.collections.Collection` (and thus both 2D
-`~matplotlib.axes.Axes.scatter` and 3D
-`~mpl_toolkits.mplot3d.axes3d.Axes3D.scatter`) no
-longer implicitly flattens its offsets.  As a consequence, ``scatter``'s ``x``
-and ``y`` arguments can no longer be 2+-dimensional arrays.
+Functions removed from the `lines` module
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+The :mod:`matplotlib.lines` module no longer imports the
+``pts_to_prestep``, ``pts_to_midstep`` and ``pts_to_poststep``
+functions from :mod:`matplotlib.cbook`.
 
-Deprecation of the former validators for ``contour.negative_linestyle``
------------------------------------------------------------------------
 
+PDF backend functions
+~~~~~~~~~~~~~~~~~~~~~
 
-The former public validation functions ``validate_negative_linestyle``
-and ``validate_negative_linestyle_legacy`` will be deprecated in 2.1 and
-may be removed in 2.3. There are no public functions to replace them.
+The methods ``embedTeXFont`` and ``tex_font_mapping`` of
+:class:`matplotlib.backqend_pdf.PdfFile` have been removed.  It is
+unlikely that external users would have called these methods, which
+are related to the font system internal to the PDF backend.
 
-::
 
-    grid.linestyle             : (1, 3)   # loosely dotted grid lines
-    contour.negative_linestyle : dashdot  # previously only solid or dashed
+matplotlib.delaunay
+~~~~~~~~~~~~~~~~~~~
 
+Remove the delaunay triangulation code which is now handled by Qhull
```

Comment:
Since I can't comment on it, the other instance of `qhull` should be capitalized to match.